SeriesMaster-Frontend/App/api/expoTokens.js

```diff
@@ -0,0 +1,16 @@
+import tokenStorage from "../auth/tokenStorage";
+import api from "./api";
+
+export async function postExpoTokens(expoToken) {
+  const token = await tokenStorage.getToken();
+  const { userId } = await tokenStorage.getUser();
+
+  await fetch(`${api.server}/api/users/pushtoken`, {
```

Comment:
You had to update all your fetches because you change the server address.
Maybe a helper function would be useful here:

```suggestion
  await fetchApi('/api/users/pushtoken', {
```

```js
function fetchApi(url, options) {
  return fetch(`${api.server}${url}`, options);
}
```

Reply:
That's a great suggestion. I will try to implement it, when I refactor my code next week.